swarmdriver.go

```diff
@@ -193,6 +193,9 @@ func (factory *swarmDriverFactory) Create(ctx context.Context, parameters map[st
 		if err != nil {
 			return nil, fmt.Errorf("Create: failed to create cached lookuper publisher: %v", err)
 		}
+		cache := initCache(ctx, clp, store)
```

Comment:
We dont need this cache. Just add logs in the initCache function. No need to return this or print it.